projects/knora-ui/src/lib/viewer/pipes/knoradate.pipe.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { KnoraDatePipe } from './knoradate.pipe';
+
+describe('KnoradatePipe', () => {
+  it('create an instance', () => {
+    const pipe = new KnoraDatePipe();
+    expect(pipe).toBeTruthy();
+  });
```

Comment:
Could you provide a minimal test case?

Reply:
done in 37865b7

Reply:
looks fine, just don't forget about the calendar.

Reply:
for sure, it'll be handled in another PR pertaining to just the custom pipe